app/assets/javascripts/redux/modules/models/brand.js

```diff
@@ -51,11 +71,26 @@ export const fetchBrandData = () => {
       BRAND_REQUEST_FAIL,
     ],
   });
-};
+}
 
 export function getBrandData() {
   return getDataFor(
     'brand',
     fetchBrandData
   );
 }
+
+export const updateBrandTwitterAdminEmail = (email) => {
+
```

Comment:
This action doesn't get dispatched when calling `this.props.dispatch(updateBrandTwitterAdminEmail(form))` from `AccountInfoForm`. Been messing with this for some time now and no clue bro!